libs/experimental/langchain_experimental/language_detector/lang_detect.py

```diff
@@ -0,0 +1,34 @@
+from typing import List, Tuple
+
+from langchain_experimental.language_detector.base import LanguageDetectorBase
+
+try:
+    import langdetect
```

Comment:
I think people from langchain team want to have failure on use, but no error on import so it does not fail in tests when modules are importer.